docs/source/howto/installation.rst

```diff
@@ -314,7 +314,45 @@ Tuning performance
 
 AiiDA supports running hundreds of thousands of calculations and graphs with millions of nodes.
 However, optimal performance at that scale might require some tweaks to the AiiDA configuration to balance the CPU and disk load.
-Here are a few general tips that might improve the AiiDA performance:
+
+Here are a few tips for tuning AiiDA performance:
+
+
+    .. dropdown:: Increase the number of daemon workers
+
+        By default, the AiiDA daemon only uses a single worker, i.e. a single operating system process.
+        If ``verdi daemon status`` shows the daemon worker constantly at high CPU usage, you can use ``verdi daemon incr X`` to add ``X`` parallel daemon workers.
+
+        Keep in mind that other processes need to run on your computer (e.g. rabbitmq, the PostgreSQL database, ...), i.e. you likely want to stop increasing the number of workers *before* you reach the number of cores of your CPU.
```

Comment:
```suggestion
        Keep in mind that other processes also need to run on your computer (e.g. rabbitmq, the PostgreSQL database and any other programs you have running), therefore you probably do not want to have the same amount or even more workers than your machine has CPUs.
```

Reply:
thanks, happy to discuss but here I prefer my version